anki/template/template.py

```diff
@@ -200,20 +205,27 @@ def render_unescaped(self, tag_name=None, context=None):
 
     def clozeText(self, txt, ord, type):
         reg = clozeReg
-        if not re.search(reg%ord, txt):
+        if not re.search(reg % ord, txt):
             return ""
+
         def repl(m):
             # replace chosen cloze with type
+            noSound = removeClozeSound(m.group(3))
```

Comment:
Somone learning a couple of verses from a song (probably not the most clever way of learning, but why not): 
Field: {{c1::[sound:part1.mp3]}}{{c2::[sound:part2.mp3]}}
That note can't be used any more, because that's the entire point of having the sound on the question (and on the answer). The limitation is pretty arbitrary, imo.